lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -388,6 +433,12 @@ def framework_output_paths(framework_input_paths)
                                       end
               [framework_output_path, *bcsymbol_output_paths]
             end.compact.uniq
+            # Static xcframeworks are not copied to the build dir
+            # so only include dynamic artifacts that will be copied to the build folder
+            xcframwork_paths = xcframeworks.select { |xcf| xcf.build_type.dynamic_framework? }.map do |xcframework|
```

Comment:
`select(&:build_type.dynamic_framework?)`?

Reply:
didn't realize you could go multiple levels down with that syntax. nice

Reply:
does this actually work? getting errors `NoMethodError: undefined method `dynamic_framework?' for :build_type:Symbol`

Reply:
oh nope that was my bad sorry. The way it is its fine.